float64[128] eigenface